README.md

```diff
@@ -90,12 +90,16 @@ A neat maven plugin for prettier-java was made by developers from HubSpot. \
 It is currently available at:
```

Comment:
If we no longer point where the plugin is available at but rather the minimal configuration we should change this text accordingly.

Reply:
I've updated the text to address this comment. Let me know what you think.

Reply:
LGTM !